src/app/modules/main/chat_section/io_interface.nim

```diff
@@ -94,7 +94,7 @@ method onChatUnmuted*(self: AccessInterface, chatId: string) {.base.} =
 method onMarkAllMessagesRead*(self: AccessInterface, chatId: string) {.base.} =
   raise newException(ValueError, "No implementation available")
 
-method onContactAccepted*(self: AccessInterface, publicKey: string) {.base.} =
+method onContactAdded*(self: AccessInterface, publicKey: string) {.base.} =
```

Comment:
I renamed the function, since it is called when a contact is added both when we accept and when we send the request, so it's not only when we accept.